doc/source/conf.py

```diff
@@ -90,6 +90,10 @@
 # The current released version
 rst_prolog = """
 .. |current_flwr_version| replace:: 1.10.0
```

Comment:
```suggestion
.. |stable_flwr_version| replace:: 1.10.0
```
I think using this term prepares us for the `1.11.0` release since `latest` can be misinterpreted as `unstable`.

Reply:
Good point! I'm happy to use `stable` here. @charlesbvll wdyt?